services/QuillLMS/app/models/stripe_integration/subscription.rb

```diff
@@ -2,6 +2,12 @@
 
 module StripeIntegration
   class Subscription < SimpleDelegator
+    def stripe_cancel_at_period_end
+      return nil if stripe_invoice_id.nil?
```

Comment:
I don't think this needs to return a value, e.g.
```suggestion
      return if stripe_invoice_id.nil?
```
(Side note, this is a minor change that would be caught by Sorbet/typing).